src/parser/youtube/VideoInfo.ts

```diff
@@ -29,6 +29,8 @@ import type { XMLDocument } from 'linkedom/types/xml/document';
 import type { Element } from 'linkedom/types/interface/element';
 import { getStringBetweenStrings, InnertubeError, streamToIterable } from '../../utils/Utils';
 import type { Node } from 'linkedom/types/interface/node';
+import { ReadableStream } from 'stream/web';
```

Comment:
This import should not be placed here, instead add it in the index.ts file and polyfill the globalThis ReadableStream here: https://github.com/LuanRT/YouTube.js/blob/main/index.ts

Reply:
This change will break Deno and browser support if added in this file.

Reply:
thank you for that info. I guess I still have a lot to learn. ☺️